pep-0517.txt

```diff
@@ -218,6 +218,23 @@ metadata. The directory passed in by the build frontend MUST be
 identical to the directory created by ``get_wheel_metadata``,
 including any unrecognized files it created.
 
+::
+
+  def prepare_build_files(build_directory, config_settings):
+      ...
+
+Must copy or create any files needed to build this package into
+``build_directory``. For instance, it is expected  that ``pyproject.toml`` will
+exist in the root of this directory after this hook runs. For tools such as
+`setuptools_scm <https://github.com/pypa/setuptools_scm>`_, this may include
+extracting some information from a version control system.
+The ``build_wheel`` hook will subsequently be run from the ``build_directory``
+populated by this hook.
```

Comment:
+1 for stating explicitly that `pyproject.toml` must be exported unchanged, and that the build output should be the same regardless of whether or not the frontend actually calls this hook.